packaging/hudi-hadoop-mr-bundle/pom.xml

```diff
@@ -29,6 +29,7 @@
   <properties>
     <checkstyle.skip>true</checkstyle.skip>
     <main.basedir>${project.parent.basedir}</main.basedir>
+    <parquet.version>${hive.parquet.version}</parquet.version>
```

Comment:
shouldn't this be put under `hudi-hadoop-mr/pom.xml` instead ?

Reply:
parquet-avro is defined as `provided` scope in parent pom (hudi-hadoop-mr/pom.xml -> hudi/pom.xml). it needs to be defined here for the compile time version to take effect.

Reply:
As discussed with @xushiyan this will make the packaging inconsistent. With this change, 
1. `hudi-hadoop-mr` will have parquet-avro version 1.12
2. `hudi-hadoop-mr-bundle` will have version 1.10
3. `hudi-spark3-bundle` will have version 1.12
if all of them are in the classpath, there could be conflict.

Reply:
For spark3.2, parquet version 1.12 is a must. 
@RexXiong Can you run `mvn dependency:tree` for all the above modules with this change and confirm the versions of parquet-avro in each of them? Our goal is to be as consistent as possible. perhaps, we could create a separate profile for hive?

Reply:
@codope The results of `mvn dependency:tree` are consistent with the results you discussed, so for classpath consistency, I agree with the idea of ​​creating a separate profile for hive.

Reply:
As discussed with @xushiyan we tend to shade the parquet-*(include parquet-avro,parquet-column,parquet-common...) within hadoop-mr-bundle, because increasing the hive profile also increases the complexity for users, @codope